tests/workflows/basic_tool_calling_node/workflow.py

```diff
@@ -0,0 +1,88 @@
+from vellum.client.types.chat_message_prompt_block import ChatMessagePromptBlock
+from vellum.client.types.function_definition import FunctionDefinition
+from vellum.client.types.plain_text_prompt_block import PlainTextPromptBlock
+from vellum.client.types.rich_text_prompt_block import RichTextPromptBlock
+from vellum.client.types.variable_prompt_block import VariablePromptBlock
+from vellum.workflows.nodes.experimental.tool_calling_node import ToolCallingNode
+from vellum.workflows.workflows.base import BaseWorkflow
+
+
+# A simple tool definition
+def get_current_weather():
```

Comment:
this function will literally get the current weather. it should have `location: str` as a parameter and do a mock api call to return some reasonable response